test/correctness/atomics.cpp

```diff
@@ -1236,14 +1236,12 @@ int main(int argc, char **argv) {
         test_all<uint64_t>(Backend::CUDA);
         test_all<int64_t>(Backend::CUDA);
         test_all<double>(Backend::CUDA);
-#if LLVM_VERSION >= 90
         test_all<uint32_t>(Backend::CUDAVectorize);
```

Comment:
This is going to cause build failures. In master LLVM_VERSION is not defined for the tests, so these tests silently don't run. However these tests are also broken. I have fixed this in the atomic_vectorization branch

Reply:
...ok then :-)

Reply:
Sorry ; (